pkg/clusters/addons/kong/kong.go

```diff
@@ -29,6 +30,24 @@ const (
 	AddonName clusters.AddonName = "kong"
 
 	httpPort = 80
+
+	// DefaultEnterpriseImageRepo default kong enterprise image
+	DefaultEnterpriseImageRepo = "kong/kong-gateway"
+
+	// DefaultEnterpriseImageTag latest kong enterprise image tag
+	DefaultEnterpriseImageTag = "2.5.0.0-alpine"
+
+	// KongLicenseSecretName is the kong license data secret name
+	KongLicenseSecretName = "kong-enterprise-license"
+
+	// EnterpriseLicense is the kong enterprise test license
+	EnterpriseLicense = "KONG_ENTERPRISE_LICENSE"
```

Comment:
I mentioned this previously but I must reiterate that the addon code shouldn't be dealing with ENV vars (calling code can, like CLI or tests) and instead we should only be taking the license contents as function/method args within this package.

```suggestion
```

Reply:
👍 Yes, the function was implemented https://github.com/Kong/kubernetes-testing-framework/pull/104/files#diff-7cc1dbd395c3c0ea6e62a7d5a3d4493f37054d7b7be23f198340cdd6a52c0be2R60. 
This default license name should be removed.

Reply:
2257e7c